ipyvizzu.py

```diff
@@ -139,24 +130,54 @@ class Chart:
 
     VIZZU = "https://cdn.jsdelivr.net/npm/vizzu@latest/dist/vizzu.min.js"
 
+    _INITIALIZE = """
+        <div id="myVizzu_{id}" style="width:{div_width}; height:{div_height};"/>
+        <script type="module">
+        import Vizzu from '{vizzu}';
+        document.chart_{id} = new Vizzu('myVizzu_{id}').initializing;
+        </script>
+    """
+
+    _SHOW = """
+        <script type="module">
+        document.chart_{id}.then(chart => {{
+            {show}.then(chart => {{
+                document.snapshot_{snapshot_id} = chart.store();
+            }});
+        }});
+        </script>
+    """
+
     def __init__(self, vizzu=VIZZU, width="800px", height="480px"):
         self._vizzu = vizzu
         self._div_width = width
         self._div_height = height
-        self._calls = []
+
+        display_html(
+            self._INITIALIZE.format(
+                id=id(self),
+                vizzu=self._vizzu,
+                div_width=self._div_width,
+                div_height=self._div_height,
+            ),
+            raw=True,
+        )
 
     def feature(self, name, value):
-        self._calls.append(Feature(name, value))
+        return self._show(Feature(name, value).dump())
 
     def animate(self, *animations: Animation, **options):
         """
-        Register new animation.
+        Show new animation.
         """
         if not animations:
             raise ValueError("No animation was set.")
 
         animation = self._merge_animations(animations)
-        self._calls.append(Animate(animation, options))
+        return self._show(Animate(animation, options).dump())
```

Comment:
@veghdev I gave a tough to the problem you mentioned, that jupyter prints out not used return values. I think it would be better just to store the last snapshot id in the class instead of returning it from the animate function, and providing a getter method for it. Maybe it could be called store(), which is a bit misleading, since the state is already stored, but from the user's point of view it would be logical, since they will use the returned snapshot id in the restore() method.